.github/pull_request_template.md

```diff
@@ -1,39 +1,42 @@
+## Type Of Change
+
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Any modification in the test cases
+- [ ] Any modification in the documentation
+
 ### Checklist:
 
 * [ ] I have read and followed the [contribution guideline](https://github.com/axem-solutions/.github/blob/main/CONTRIBUTING.md).
-* [ ] I have checked to ensure there aren't other open Pull Requests for the same update/change
-* [ ] I have tested these changes and added tests that prove my fix is effective or that my feature works
-* [ ] I have made corresponding changes to the documentation
-* [ ] New and existing unit tests pass locally with my changes
-* [ ] 100% unit test coverage achieved
-
+* [ ] I have checked to ensure there aren't other open Pull Requests for the same update/change.
+* [ ] Every new modification in the production code has been tested if there is any.
```

Comment:
I'm not sure I understand this sentence

Reply:
I can rephrase it like:
If there is any modification in the production code it has been tested.